core/src/main/scala/io/finch/request/package.scala

```diff
@@ -198,14 +202,15 @@ package object request extends LowPriorityImplicits {
   object RequiredParam {
 
     /**
-     * Creates a ''RequestReader'' that reads a required string ''param''
-     * from the request or raises an exception when the param is missing or empty.
+     * Creates a [[RequestReader]] that reads a required string ''param'' from the request or raises a [[NotPresent]]
```

Comment:
Does this syntax now work in recent Scala versions? Older versions required the use of FQN with the bracket syntax.